picoCTF-web/web/classroom.html

```diff
@@ -181,7 +179,7 @@ <h4>Exporting Data</h4>
             </div>
           </div>
           <% _.each(_.sortBy(teams, "team_name"), function(team) {
-            members = team.members.map(function(member){ return member.username }).join(", ") %>
+            members = team.members.map(function(member){ return member.firstname + " " + member.lastname }).join(", ") %>
```

Comment:
This (and in `classroom.coffee` below) is where we should introduce a function that *uniformly* returns a human-friendly string to identify a member regardless of whether we collect names or not. We know a member has a `username` and optionally a `firstname` and a `lastname`. The function would return `"({{firstname}} {{lastname}} ({{username}})"` if we have the name, or `"{{username}}"` if we don't. That way, we won't need to change any code until we need to deal with languages that put the last name in the front.

Case 1: "John Doe (jdoe)"
Case 1 missing first name: "Doe (jdoe)"
Case 1 missing last name: "John (jdoe)"
Case 2: "jdoe"

Reply:
I think we'd only have case 1 and 2, not missing only 1b) fname or 1c) lname. Those are required fields currently (our 2018 competition replaced them with `-`)

Reply:
Addressed by d9a437e